launcher/minecraft/skins/SkinModel.cpp

```diff
@@ -23,10 +23,14 @@
 #include "FileSystem.h"
 #include "Json.h"
 
-static QImage improveSkin(const QImage& skin)
+static QImage improveSkin(const QImage& _skin)
 {
+    auto skin = _skin;
+    if (skin.format() != QImage::Format_RGBA8888) {
+        skin = skin.convertToFormat(QImage::Format_RGBA8888);
+    }
     if (skin.size() == QSize(64, 32)) {  // old format
-        QImage newSkin = QImage(QSize(64, 64), skin.format());
+        auto newSkin = QImage(QSize(64, 64), skin.format());
```

Comment:
```suggestion
        auto newSkin = QImage(QSize(64, 64), QImage::Format_RGBA8888);
```
It seems the skin from the issue was in the "old format". I *assume* this is a part of it since we haven't gotten other reports, so I'm not sure why we would always convert to this format